test/e2e/scheduling/preemption.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
I guess because serial test does not enable alpha feature gate for preemption, CI will not actually run this test?

Reply:
I think you are right. I will look into this.

Reply:
Yeah you should probably ask on the [kubernetes-sig-testing](https://groups.google.com/forum/#!forum/kubernetes-sig-testing) mailing list. They might tell you to create a separate Jenkins job just for the e2e tests for this feature while it is in alpha, and then merge it back into regular serial once it goes to beta and is no longer flag-gated.

Reply:
I sent the email. Will make the changes once I learn how to do it.

Reply:
There already is a gce and gke alpha suite.  Just tag the tests with "[Feature:Preemption]" and modify the [config](https://github.com/kubernetes/test-infra/blob/master/jobs/config.json) for the alpha suites to add the tag.

Reply:
Yes, that's what I just did: https://github.com/kubernetes/test-infra/pull/4285